### PR TITLE
Feature/attrition

### PR DESCRIPTION
**Pull request by cschaerfe**
Added attrition features with new API including `get_attritition`, `apply_attrition` and `plot`. Tests are provided for get and apply, but not yet for plotting.

### CODE REVIEW

**Comment by bailliem** [comment]
@cschaerfe  -- I've resolved the merge conflicts. There was an update to plot.R that was not reflected in your branch.

**Comment by cschaerfe** [comment]
Testing helps! Found one more bug that I fixed in cases where one would use a concatenated filter using OR (e.g., "AGE > 61 | SEX == 'M'" followed by additional filters

**Review by bailliem — APPROVED** [review]

File: DESCRIPTION [hunk 1]
```diff
@@ -33,8 +33,7 @@ Suggests:
     forcats,
     glue,
     here,
-    stringr,
-    dplyr
+    stringr
```
Comment (SHAESEN2) [inline DESCRIPTION:36]: stringr is a package that we don't really need. These are wrappers around base sub gsub and trimws

**Review by SHAESEN2 — APPROVED** [review]
Do we need stringr?

File: DESCRIPTION (hunk 1 above)
Comment (cschaerfe) [reply DESCRIPTION:36]: Good point. I think I used it, but we can consider taking it out again, if we want to

Comment (cschaerfe) [reply DESCRIPTION]: It only shows up under suggests, so I guess we can keep it here and then clean up description before the release?

**Review by rebecca-albrecht — APPROVED** [review]

**Pull request merged**